api/Navigator.json

```
@@ -575,7 +575,8 @@
             },
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": "17",
+              "notes": "Badging is supported for web apps saved to the Dock in Safari 17 on the macOS Sonoma beta"
```

I assume that it's not just apps added to the dock, but any installed PWA (including in the Launchpad).  (If it really is just accessible for dock apps...well, that'd be pretty bizarre.  :P)  Let's simplify the note everywhere to the following:

```suggestion
              "notes": "Badging is supported for installed web apps on macOS Sonoma and higher."
```